.github/scripts/check_path_length.py

```
@@ -0,0 +1,75 @@
+DEFAULT_BUDGET = 171
+
+
+def vault_paths():
+    """Every path git would ship: tracked files plus new files not yet added.
+
+    Untracked files count because a note that has just been written, and not
+    yet staged, is exactly the case this check exists to catch. Ignored files
+    are excluded, so scratch files do not trip it.
+    """
+    out = subprocess.run(
+        ["git", "ls-files", "-z", "--cached", "--others", "--exclude-standard"],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)